integration/container/copy_test.go

```diff
@@ -15,14 +15,13 @@ import (
 	"github.com/docker/docker/integration/internal/container"
 	"github.com/docker/docker/pkg/jsonmessage"
 	"github.com/docker/docker/testutil/fakecontext"
-	"gotest.tools/v3/assert"
-	is "gotest.tools/v3/assert/cmp"
```

Comment:
Looks like a bad rebase; should be including the `v3`

Reply:
Yea. Noticed couple of other errors too. Playing same time with #40599 as trying to make tests really failing first...